packages/hawtio/src/plugins/auth/oidc/index.ts

```diff
@@ -0,0 +1,16 @@
+import { HawtioPlugin } from '@hawtiosrc/core'
+import { helpRegistry } from '@hawtiosrc/help/registry'
+import help from './help.md'
+import { oidcService } from './oidc-service'
+import { Logger } from '@hawtiosrc/core'
+
+const pluginName = 'hawtio-oidc'
+const log = Logger.get(pluginName)
+
+const oidc: HawtioPlugin = () => {
+  log.info('Loading oidc() plugin')
```

Comment:
that's something we don't normally do. we can check whether a plugin is loaded or not by checking the hawtio core log.

Reply:
done